docs/intro/index.md

```diff
@@ -7,105 +7,46 @@ altRoutes:
-> if a distributed system is really a single system, then the processes must be
-> synchronized in some way. Conceptually, the easiest way to synchronize
-> processes is to get them all to do the same thing at the same time. Therefore,
-> our method is used to implement a kernel that performs the necessary
-> synchronization--for example, making sure that two different processes do not
-> try to modify a file at the same time. Processes might spend only a small
-> fraction of their time executing the synchronizing kernel; the rest of the
-> time, they can operate independently--e.g., accessing different files. This is
-> an approach we have advocated even when fault-tolerance is not required. The
-> method's basic simplicity makes it easier to understand the precise properties
-> of a system, which is crucial if one is to know just how fault-tolerant the
-> system is.
-> [\[L.Lamport (1984)\]](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.71.1078)
+Building the best user experience in an application is a top priority for
+developers. With blockchains, the user experience is often limited by the
+underlying technology, causing slow response times and high fees. The low fees
+and 400ms confirmation times enable developers to build applications that are
+user friendly and accessible to everyone.
 
-Furthermore, and much to our surprise, it can be implemented using a mechanism
-that has existed in Bitcoin since day one. The Bitcoin feature is called
-`nLocktime` and it can be used to postdate transactions using block height
-instead of a timestamp. As a Bitcoin client, you would use block height instead
-of a timestamp if you don't rely upon the network. Block height turns out to be
-an instance of what's being called a Verifiable Delay Function in cryptography
-circles. It's a cryptographically secure way to say time has passed. In Solana,
-we use a far more granular verifiable delay function, a SHA 256 hash chain, to
-checkpoint the ledger and coordinate consensus. With it, we implement Optimistic
-Concurrency Control and are now well en route towards that theoretical limit of
-710,000 transactions per second.
+## Solana Features
 
-## Documentation Overview
+| Feature                      | Description                                                                                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| On-chain Program Development | The ability of developing and deploying programs on-chain. Users can interact with these programs permissionlessly, removing the need for middleman servers to build applications.                                           |
```

Comment:
something about the current working just seems off to me. I can't place exactly why though:

```suggestion
| On-chain Program Development | The ability to develop and deploy programs on-chain. Users can interact with these programs permissionlessly, removing the need for middleman servers to build applications.                                           |
```